voice/transfer-call-inline-ncco.py

```diff
@@ -1,40 +1,42 @@
-# Import dependencies
-import nexmo
-import os
+#Import the dependencies
+import nexmo, os
 from os.path import join, dirname
 from dotenv import load_dotenv
+from pprint import pprint
 
-# Load the environment
-envpath = join(dirname(__file__), './.env')
+#Load the environment
+envpath = join(dirname(__file__),'./.env')
 load_dotenv(envpath)
 
-# Init the client
+#Init the client
 print(os.getenv("NEXMO_PRIVATE_KEY"))
 client = nexmo.Client(
-    application_id=os.getenv('NEXMO_APPLICATION_ID'),
-    private_key=os.getenv("NEXMO_PRIVATE_KEY")
+    application_id = os.getenv('NEXMO_APPLICATION_ID'),
```

Comment:
I think this should be in the `example.env` file to prompt people to set this value

Reply:
(also the `NEXMO_PRIVATE_KEY`)

Reply:
Corrected. And Adding to example.env